ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracing_shim/__init__.py

```diff
@@ -22,20 +88,56 @@
 
 
 def create_tracer(otel_tracer):
+    """Creates a :class:`TracerShim` object from the provided OpenTelemetry
+    :class:`opentelemetry.trace.Tracer`.
+
+    The returned :class:`TracerShim` is an implementation of
+    :class:`opentracing.Tracer` using OpenTelemetry under the hood.
+
+    Args:
+        otel_tracer: A :class:`opentelemetry.trace.Tracer` to be used for
+            constructing the :class:`TracerShim`. This tracer will be used
+            to perform the actual tracing when user code is instrumented using
+            the OpenTracing API.
+
+    Returns:
+        The created :class:`TracerShim`.
+    """
+
     return TracerShim(otel_tracer)
 
 
 class SpanContextShim(opentracing.SpanContext):
+    """Implements :class:`opentracing.SpanContext` by wrapping a
+    :class:`opentelemetry.trace.SpanContext` object.
+
+    Args:
+        otel_context: A :class:`opentelemetry.trace.SpanContext` to be used for
+            constructing the :class:`SpanContextShim`.
+    """
+
     def __init__(self, otel_context):
         self._otel_context = otel_context
 
     def unwrap(self):
-        """Returns the wrapped OpenTelemetry `SpanContext` object."""
+        """Returns the wrapped :class:`opentelemetry.trace.SpanContext`
+        object.
+
+        Returns:
+            The :class:`opentelemetry.trace.SpanContext` object wrapped by this
+            :class:`SpanContextShim`.
+        """
 
         return self._otel_context
 
     @property
     def baggage(self):
+        """Implements the `baggage` property from the base class.
```

Comment:
The dark side of the "any" role is that this "baggage" links to the function itself, you have to use double-backticks to get it to render as regular code.

Reply:
Thanks, good catch. I hope I fixed all of these now.